development/site-components/navbar.pug

```diff
@@ -0,0 +1,26 @@
+html
+	//- <!-- Navbar -->
+	nav(class="navbar navbar-default navbar-fixed-top")
+		div.container-fluid
+			div.navbar-header
+				button.navbar-toggle(type="button" data-toggle="collapse" data-target="#myNavbar")
+					span.icon-bar
+					span.icon-bar
+					span.icon-bar
+				a.navbar-brand(href="/index.html") MSPUDDING
+
+			div(class="collapse navbar-collapse" id="myNavbar")
+				ul(class="nav navbar-nav navbar-right")
+					li
+						a(href="/about.html") ABOUT
+					li
+						a(href="/blog/proteinpancakes.html") BLOG
+
+						//-Dropdown button code if I need it in the future
+	//- <!-- 				<li class="dropdown">
```

Comment:
This looks outdated, I would delete it. We'll likely be making some major improvements to the over UI anyway.